opennms-base-assembly/src/main/filtered/bin/vmwareconfigbuilder

```diff
@@ -11,12 +11,13 @@ SLF4J_L4J_JAR=`ls -1 "$OPENNMS_HOME"/lib/log4j-over-slf4j-*.jar | head -n 1`
 DOM4J_JAR=`ls -1 "$OPENNMS_HOME"/lib/dom4j-*.jar | head -n 1`
 COMMONS_CLI_JAR=`ls -1 "$OPENNMS_HOME"/lib/commons-cli-*.jar | head -n 1`
 CORE_JAR=`ls -1 "$OPENNMS_HOME"/lib/org.opennms.core.lib-*.jar | head -n 1`
+JDK9_JAXB=`ls -1 "$OPENNMS_HOME"/lib/jdk9plus/jaxb-runtime-*.jar | head -n 1`
```

Comment:
Does this still work under JDK8, or does it make things weird there? (Do we have to make it conditional like we do in the main opennms script?)

Reply:
This works fine. The other script vmwarecimquery already used this construct.

Reply:
+1